GameStore.Services/Services/IProducerService.cs

```diff
@@ -0,0 +1,14 @@
+﻿using GameStore.Domains.Domain;
+using System;
```

Comment:
Remove unnecessary usings. Please, apply this to each class in further.

Reply:
Fixed